### DISCUSSION TITLE
DNA-protein sims in MARTINI 3

### DISCUSSION THREAD

**Discussion by agoliaei** [marrink-lab/vermouth-martinize#568] [Q&A] [answered]
Hi,
Thank you all for sharing your work with the community!
I think I have confused myself a little bit, so I decided to reach out and ask for help. Is it possible to do CGed DNA-protein simulations using MARTINI 3 parameters? I see there is an itp file in here: martini_v300.zip, but I don't see any mappings. 
If this is not the right place for this question please tell me.
I appreciate your help.

**Comment by pckroon** [comment]
Unfortunately, afaik there are 2 issues: 
1) there are no links defining the bonds/connections/parameters between the protein and DNA parts. This does not matter if the same system contains DNA and protein molecules separately, and if you have a DNA/protein chimera you can always add the links yourself.
2) The CLI only lets you specify elastic network parameters for the entire system, and as far as I know they should be different for DNA and proteins. 

@fgrunewald do you know of a workaround?

**Reply by fgrunewald** [reply, marked as answer]
@agoliaei Martini3 DNA is currently not available; we're working hard on finalizing the model but I cannot make any predictions when this model is released.

**Reply by agoliaei** [reply]
Thanks so much for clarifying. I appreciate it.
